Uno.Gallery/Uno.Gallery.UITest/Given_RatingControl.cs

```diff
@@ -20,8 +20,22 @@ public void RatingControl()
 			var Rating = new QueryEx(x => x.All().Marked("Rating_Fluent"));
 			var Minus = new QueryEx(x => x.All().Marked("Minus_Fluent"));
 			var value = new QueryEx(x => x.All().Marked("Value_Fluent"));
+			var Plus = new QueryEx(x => x.All().Marked("Plus_Fluent"));
+
+
+			Minus.Tap();
+			Assert.AreEqual("Value:  2", value.GetDependencyPropertyValue<string>("Text"));
+			var clear = new QueryEx(x => x.All().Marked("Clear_Fluent"));
+			Plus.Tap();
+			Assert.AreEqual("Value:  2", value.GetDependencyPropertyValue<string>("Text"));
```

Comment:
Instead of asserting the value of the TextBlock.Text, we should assert that the Value property of the RatingControl itself.

So give the RatingControl an automationid and then  `Assert.AreEqual(2, FluentRatingControl.GetDependencyPropertyValue<double>("Value"));

Reply:
@kazo0  there is no FluentRatingControl , didnt you mean `ratingHolder`
https://github.com/unoplatform/Uno.Gallery/blob/cb187874384d002034649ba67a57a13c7e94df3c/Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/RatingControlSamplePage.xaml#L35C22-L35C34